### PR TITLE
Collapse fields on vulnerability page when it's too long

### PR DESCRIPTION
**Pull request by ZhangChen199102**
Issue: #2216

Result:
May-21-2024 15-13-51

Does not render a toggle button if it's a short list:
image

### CODE REVIEW

File: gcp/appengine/frontend3/src/templates/vulnerability.html
Comment (ZhangChen199102) [inline gcp/appengine/frontend3/src/templates/vulnerability.html:41]: We can make any `<ul>...</ul>` collapsible on this page by adding `expandible-list` class.

File: gcp/appengine/frontend3/src/templates/vulnerability.html
Comment (ZhangChen199102) [inline gcp/appengine/frontend3/src/templates/vulnerability.html:249]: Currently it shows at most 5 items by default, but we can change this number later.

Comment (andrewpollock) [reply gcp/appengine/frontend3/src/templates/vulnerability.html:249]: Rather than hard-coding a number, is it possible to take the size of the window into consideration? I think it's important for someone viewing a vulnerability record to be able to visually reason about all of the affected packages in a single page (where the window is reasonably large)

Comment (andrewpollock) [reply gcp/appengine/frontend3/src/templates/vulnerability.html:249]: This was more about expanding the affected packages, than collapsing the references (which I definitely think should be collapsed by default if more than say 5)

**Review by andrewpollock — APPROVED** [review]
LGTM, let's submit after this week's release to review in staging

**Comment by ZhangChen199102** [comment]
Thanks @andrewpollock, can you please help me to merge this PR, I just realised that I lost the write access

**Pull request merged**
